<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dormmanage</class>
 <widget class="QWidget" name="dormmanage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>学生的住宿情况</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>请输入你要查找的学号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dorm_inputlineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="dorm_queryButton">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>寝室号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>学号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>姓名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>性别：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>床位：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="dormqingshi_numlineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="dorm_outIDlineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="dorm_namelineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="dorm_sexlineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="dorm_bednumlineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="dorm_addButton">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dorm_deleteButton">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dorm_backButton">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="see_all_dorm_Button">
     <property name="text">
      <string>查看所有学生的住宿</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
